.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_style = space
```

Comment:
Just remove this whole file. We should keep to Gatsby's style while developing it but we shouldn't push that style on people's sites.